### PR TITLE
Darr

### PR DESCRIPTION
**Pull request by gbeckers**

### CODE REVIEW

File: recipes/darr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "darr" %}
+{% set version = "0.1.7" %}
```
[the remark this answers is not in the document]
Comment (gbeckers) [reply recipes/darr/meta.yaml]: Done!

File: recipes/darr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy >=1.12
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```
[the remark this answers is not in the document]
Comment (gbeckers) [reply recipes/darr/meta.yaml]: There was a reason for >= numpy 1.12, but I forgot what is was.... Perhaps some tests failed on windows. I will look into this first.

File: recipes/darr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  imports:
+    - darr
+
+about:
+  home: https://github.com/gbeckers/darr
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
[the remark this answers is not in the document]
Comment (gbeckers) [reply recipes/darr/meta.yaml]: The license file that is available with the recipe is also available in the pypi package. I am new to conda packaging. Should it just be present in one place?

File: recipes/darr/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (gbeckers) [reply recipes/darr/meta.yaml]: I think that lower versions simply were not explicitly tested, but there is no specific reason that they shouldn't. I removed the version requirement.

File: recipes/darr/meta.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jakirkham) [reply recipes/darr/meta.yaml]: Yeah let's grab it from the PyPI package. Would make this change.

```suggestion
  license_file: LICENSE'
```

File: recipes/darr/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
```
Comment (jakirkham) [inline recipes/darr/meta.yaml]: Are we linking to NumPy? If not, I think we can drop this.

File: recipes/darr/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jakirkham) [reply recipes/darr/meta.yaml]: Yeah maybe change to this then

```suggestion
    - numpy >=1.12
```

**Comment by jakirkham** [comment]
Sorry for the lag here @gbeckers. I added a few comments with suggestions. Let's check the state of CI after those are made.

**Comment by jakirkham** [comment]
Thanks @gbeckers ! 😀

Could you please also merge with the latest upstream `master`? That will clear out some CI errors due to old configs and such 🙂

File: recipes/darr/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+test:
+  imports:
+    - darr
+
+about:
+  home: https://github.com/gbeckers/darr
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A Python science library for memory-mapped numeric arrays, based on a format that is self-explanatory and tool-independent.'
```
Comment (jakirkham) [inline recipes/darr/meta.yaml:35]: Would wrap this as well.

```suggestion
  summary: >-
    A Python science library for memory-mapped numeric arrays, based on
    a format that is self-explanatory and tool-independent.
```

File: recipes/darr/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  description: |
+    Darr is a Python science library that enables you to work efficiently with
+    disk-based numeric arrays without depending on tool-specific data formats.
+    This makes it easy to share your data with those who do not use Darr or even
+    Python. No exporting required and, as the data is saved in a self-explanatory
+    way, not much explanation required either. Tool-independent and easy access
+    to data is in line with good scientific practice as it promotes wide and
+    long-term availability, to others but also to yourself.
+  doc_url: http://darr.readthedocs.io/
```
Comment (jakirkham) [inline recipes/darr/meta.yaml:44]: Suggest using `https` as RTD supports it.

```suggestion
  doc_url: https://darr.readthedocs.io/
```

File: recipes/darr/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 183886142de227c8179409b8c69a5dfec0c74d467f9aa78be8224070961cad1b
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (jakirkham) [inline recipes/darr/meta.yaml:15]: We've baked these other arguments into Conda-Build to simplify the user experience. So you can just write...

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/darr/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 183886142de227c8179409b8c69a5dfec0c74d467f9aa78be8224070961cad1b
+
+build:
+  number: 0
+  skip: True  # [py<36]
```
Comment (jakirkham) [inline recipes/darr/meta.yaml:14]: Would suggest making this `noarch` to simplify the build process and make it easier for more users to install.

```suggestion
  noarch: python
```

File: recipes/darr/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
```
Comment (jakirkham) [inline recipes/darr/meta.yaml:19]: Part of the `noarch` suggestion above

```suggestion
    - python >=3.6
```

File: recipes/darr/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    - pip
+    - setuptools
+  run:
+    - python
```
Comment (jakirkham) [inline recipes/darr/meta.yaml:23]: Part of the `noarch` suggestion above

```suggestion
    - python >=3.6
```

Comment (gbeckers) [reply recipes/darr/meta.yaml]: Many thanks jakirkham for all your help!

**Comment by jakirkham** [comment]
Thanks @gbeckers! 😄

**Comment by gbeckers** [comment]
Many thanks  jakirkham for guiding me through this! Much appreciated!

**Comment by jakirkham** [comment]
Of course! Feedstock should be ready to go ( https://github.com/conda-forge/darr-feedstock ) and you should have permissions there. Packages appear to already be published. Please let me know if there’s anything else 🙂

**Comment by gbeckers** [comment]
One more question: I created a new darr version (minor) on PyPi, which was picked up by regro-cf-autotick-bot as a PR on  conda-forge / darr-feedstock (I also sucessfully created a similar PR manually, but closed it). So far so good, but I can't find any way to merge open PRs on darr-feedstock. Am I missing something? Thanks!

**Comment by jakirkham** [comment]
Should just be a matter of accepting the invite to join conda-forge 🙂

https://github.com/conda-forge/darr-feedstock/commit/9d38ace5d8ba93a1554d775aed722d2b03b7dea8#commitcomment-39020370

**Pull request merged**